src/hotspot/cpu/x86/stubGenerator_x86_64_adler.cpp

```diff
@@ -282,6 +282,7 @@ address StubGenerator::generate_updateBytesAdler32() {
 
   // continue loop
   __ movdl(xa, a_d);
+  __ vpxor(yb, yb, yb, Assembler::AVX_256bit);
```

Comment:
Thanks @sviswa7 , can you please move this to start of SLOOP1, we can then remove two [vpxors]() before the start for AVX2 and AVX3 loops. Also there was a minor typo in the comment over algorithm, attaching the change file.
[suggestion.txt](https://github.com/openjdk/jdk/files/10752579/suggestion.txt)